rspec-its.gemspec

```diff
@@ -21,9 +21,7 @@ Gem::Specification.new do |spec|
     'source_code_uri'   => 'https://github.com/rspec/rspec-its',
   }
 
-  spec.files         = `git ls-files`.split($/) - %w[cucumber.yml]
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files         = %w{LICENSE.txt} + Dir.glob("lib/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
```

Comment:
So if we go down this path, we should use `git ls-files` as the base and filter them, so that accidentally existing but not committed files don't get released in a gem.

Reply:
In fact this is from `rspec-core`:
```
spec.files            = `git ls-files -- lib/*`.split("\n")
spec.files           += %w[README.md LICENSE.md Changelog.md .yardopts .document]
spec.test_files       = []
```

Which is a lot neater!